### DISCUSSION TITLE
See space or other chars fill

### DISCUSSION THREAD

**Discussion by valix85** [janisdd/vscode-edit-csv#149] [Ideas]
Hello all, it's possible to set an option to see/highlight any empty spaces? 
I often find myself working with csv in which the columns are padded with spaces or 0s. It would be useful to be able to see that the contents of the column are not empty but have a series of spaces or 0s or other characters.

**Comment by janisdd** [comment]
Perhaps an option with a regex value that then highlights matching cells (similar to the search results)?
